src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,6 +51,25 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    swerveX = swerveSubsystem.getPose().getX();
+    
+    if (swerveX >= 0 && swerveX <= Units.feetToMeters(4)) {
+      simPitch = Units.degreesToRadians((swerveX * (22/Units.feetToMeters(4))) - 11);
+    }
+    else {
+      simPitch = 0;
+    }
+
+    SmartDashboard.putNumber("PoseX", swerveX);
```

Comment:
This is already tracked in the robot pose. Remove because its redundant

Reply:
Deleted